rq/job.py

```diff
@@ -750,4 +748,58 @@ def register_dependency(self, pipeline=None):
             connection.sadd(dependents_key, self.id)
             connection.sadd(self.dependencies_key, dependency_id)
 
+    @property
+    def dependencies_job_ids(self):
+        dependencies = self.connection.smembers(self.dependencies_key)
+        return [Job.key_for(as_text(_id))
+                for _id in dependencies]
+
+    def dependencies_are_met(
```

Comment:
Nitpick: make the args one line.

Reply:
The order of arguments should be `self`, `exclude_job_id`, `pipeline`

Reply:
:heavy_check_mark: in 24d2a399164adc6214df71f1507b84d05b8af789